fontforge/python.c

```diff
@@ -5155,6 +5155,7 @@ return( NULL );
 		if ( !c->points[nexti]->on_curve ) {
 		    SplinePointListsFree(ss);
 		    PyErr_Format(PyExc_TypeError, "In cubic splines there must be exactly 2 control points between on curve points");
+                    free(sp);
```

Comment:
This is "new", and seems to be OK, but if it's added here it makes little sense not to also add it after (new) line 5148.

Reply:
Also, move this up before the error, so that we're freeing things "together".

Reply:
Oh, maybe it's not new. These fixes tend to blend together after a while.

Reply:
In my last commit, I moved up free(sp) + added a new one just some lines before